app/not-found.jsx

```diff
@@ -0,0 +1,43 @@
+import { ogMeta, twitterMeta } from '@data/metadata'
+
+
+export async function generateMetadata() {
+  const description = `Uh-oh! It appears you've reached a digital crossroads, and the page you were seeking has gone missing. Don't worry, though; we're here to assist you in navigating back to our main content or helping you find the information you were looking for. Feel free to explore our site, or use the search bar to discover what you need. We apologize for any inconvenience and appreciate your patience.`
+  return {
```

Comment:
This is massively overkill, it's a 404 page, not a diary entry! 😅 

Please just use this component - https://www.hyperui.dev/components/application-ui/error-pages#component-1

Reply:
So remove meta tags?

Reply:
Shorten description